iroha-lib/QueryExample.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+#include <cassert>
+
+#include "model/utils/Utils.h"
+
+
+void sendSampleTransaction(const std::string& account_name,
+                           const std::string& key_path,
+                           const std::string& peer_ip,
+                           uint16_t torii_port,
+                           uint32_t quorum,
+                           const std::string& domain_id,
+                           const std::string& user_default_role,
+                           const std::string& asset_name)
+{
+    const auto tx_proto = generateTransaction(account_name,
+                                              key_path,
+                                              domain_id,
+                                              user_default_role,
+                                              asset_name);
+
+    iroha_lib::GrpcClient(peer_ip, torii_port).send(tx_proto);
+
+    const auto tx_hash = getTransactionHash(tx_proto);
+    printTransactionStatus(peer_ip, torii_port, tx_hash);
+}
+
+
+void run()
+{
+    auto account_name = "admin@test";
+    const auto key_path = "/home/laptop/qt-workspace/iroha/example";
+    const auto peer_ip = "127.0.0.1";
+    uint16_t torii_port = 50051;
+    const auto quorum = 1;
+    const auto user_default_role = "user";
+
+    sendSampleTransaction(account_name,
+                          key_path,
+                          peer_ip,
+                          torii_port,
+                          quorum,
+                          "domainsamplev4",
+                          user_default_role,
+                          "assetnamesamplev4");
+
+    uint64_t counter = 0u;
+    const auto query_proto = generateQuery(account_name, key_path, counter);
+    iroha_lib::GrpcClient(peer_ip, torii_port).send(query_proto);
+    assert(query_proto.payload().get_account_assets().account_id() == account_name);
+}
+
+
+int main()
+{
+    run();
+    return 0;
+}
```

Comment:
Really important use case of iroha library is to loop over transactions, so I suggest to add another example, which would do something like there: https://github.com/hyperledger/iroha-python/blob/0b260d480ac96ca1e33c0ea7e2a08a98e1378cc6/examples/query-transactions.py#L212
which is:
1. creating few transfer transactions in loop
2. query them displaying transactions one by one in a loop
It would be really helpful to have that kind of example:)